q2templates/templates/tabbed.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'base.html' %}
+
+{% block content %}
+{% if tabs is defined %}
+  <div class="row">
+    <div class="col-lg-12">
+      <ul class="nav nav-tabs">
+        {% for tab in tabs %}
+          <li class="{% if self._TemplateReference__context.name == tab.url %}active{% endif %}">
```

Comment:
this looks like it hurt to figure out... I assume there is no other way?

Reply:
Yeah, it isn't ideal, but I figured it was okay for now since this is primarily an "internal" tool. Another solution is to set a variable in the child templates:

```jinja
{# parent #}
<li class="{% if current_tab == tab.url %}active{% endif %}">

{# child #}
{% set current_tab = 'tab-1.html' %}
```